doc/classes/Basis.xml

```diff
@@ -187,25 +187,6 @@
 				Returns the transposed version of the matrix.
 			</description>
 		</method>
-		<method name="xform">
-			<return type="Vector3">
-			</return>
-			<argument index="0" name="v" type="Vector3">
-			</argument>
-			<description>
-				Returns a vector transformed (multiplied) by the matrix.
-			</description>
-		</method>
-		<method name="xform_inv">
-			<return type="Vector3">
-			</return>
-			<argument index="0" name="v" type="Vector3">
-			</argument>
-			<description>
-				Returns a vector transformed (multiplied) by the transposed basis matrix.
-				[b]Note:[/b] This results in a multiplication by the inverse of the matrix only if it represents a rotation-reflection.
-			</description>
-		</method>
```

Comment:
`Basis`, `Transform` and `Transform2D` all lost their `xform` and `xform_inv` methods. Bug?

Reply:
Discussed it with @reduz, this was done on purpose as these are too complex to bind in the new system, and the multiplication operators can and should be used instead:
```
xform == mat * vec
xform_inv == vec * mat
```

Reply:
I also removed `Quat.xform` which had been kept.